plugin/src/main/java/org/opensearch/ml/rest/RestMLStatsAction.java

```diff
@@ -148,6 +149,10 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
     }
 
     MLStatsInput createMlStatsInputFromRequestParams(RestRequest request) {
+
+        Set<String> mlNodeStatNames = EnumSet.allOf(MLNodeLevelStat.class).stream()
```

Comment:
Should we construct a new Set for every request?

Reply:
we can construct the new set in the class initialization. Let me do that.